src/server/object_services/object_server.js

```diff
@@ -72,8 +72,21 @@ async function create_object_upload(req) {
         encryption
     };
 
-    const lock_settings = config.WORM_ENABLED ? calc_retention(req) : undefined;
-    if (lock_settings) info.lock_settings = lock_settings;
+    if (req.rpc_params.partial_object) {
```

Comment:
remove `partial_object`? or maybe in this case we still need this as a flag to create_object_upload?

Reply:
I think we still need it.